src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -576,11 +576,11 @@ public function testResolve()
         ]);
 
         $this->assertSame(200, $response->getStatusCode());
-        $this->assertSame(200, $client->request('GET', 'http://symfony.com:8057/')->getStatusCode());
+        $this->assertSame(200, $client->request('GET', 'http://symfony.com:8057/', ['timeout' => $timeout = 0.1])->getStatusCode());
```

Comment:
That prevents from waiting 3 sec while 0.1 is enough.

Reply:
why have a timeout at all here?

Reply:
Related to https://github.com/symfony/symfony/pull/32948#discussion_r310592311 but I will remove the timeout from here and set 1 sec for the other.